src/test/rpc/LedgerRPC_test.cpp

```diff
@@ -1226,15 +1226,32 @@ class LedgerRPC_test : public beast::unit_test::suite
         // "features" is not an option supported by ledger_entry.
         Json::Value jvParams;
         jvParams[jss::api_version] = apiVersion;
-        jvParams[jss::features] = ledgerHash;
-        jvParams[jss::ledger_hash] = ledgerHash;
-        Json::Value const jrr =
-            env.rpc("json", "ledger_entry", to_string(jvParams))[jss::result];
+        {
+            jvParams[jss::features] = ledgerHash;
+            jvParams[jss::ledger_hash] = ledgerHash;
+            Json::Value const jrr = env.rpc(
+                "json", "ledger_entry", to_string(jvParams))[jss::result];
+
+            if (apiVersion < 2u)
```

Comment:
nit: why is apiVersion checked with < 2u in this file compared > 1u? seems better to keep consistent throughout codebase (if most files have already established some standard)

Reply:
In the test file, it is specifically checking results for apiVersion < 2u and apiVersion other than that (>=2) , while in non-test files, the new error codes only execute if apiVersion is greater than 1. There is a guideline already established [here](https://docs.google.com/document/d/1qEMoDyi7or-7ODEfVtnaFlcFSb0NsE2vnvEw6MIJ3Zo/edit):

Reply:
Don't have access to that doc, but thanks for elaboration. If you don't mind sharing it, my email is connorjchen2784@gmail.com

Is `> 1u` in non-test file equivalent to `>= 2u` in test file? These seem to always be equivalent expressions since it is impossible to have decimal (1.5) here, right?

Reply:
yea > 1u in non-test file is equivalent to >= 2u in test file.